workers/unity/Assets/Fps/Prefabs/ClientWorkerHandler.prefab

```diff
@@ -54,4 +54,5 @@ MonoBehaviour:
   clientWorkerPrefab: {fileID: 1638104666241782, guid: 075c14fd6c4ad514581b6aaabc0711bb,
     type: 2}
   canvasCameraObj: {fileID: 0}
-  screenUIController: {fileID: 0}
+  screenUIControllerDesktop: {fileID: 0}
+  screenUIControllerMobile: {fileID: 0}
```

Comment:
Hmm, these are old properties and it's since been switched back to just screenUIController. I'll fix this